### DISCUSSION TITLE
Can I standardize the number format?

### DISCUSSION THREAD

**Discussion by GabrielCoderz** [perspective-dev/perspective#2554] [Q&A]
I would like to standardize the numbers, for exemple: 

The number 1000000 must be always 1.000.000
But sometimes i see it with comma, like 1,000,000
Or with spaces: 1 000 000

Currently, it's possible ?

I would be very grateful to anyone who can help.

**Comment by timkpaine** [comment]
This is being done in https://github.com/finos/perspective/pull/2539

**Reply by texodus** [reply]
#2539 does not address number sep character as this is part of the Browser's locale.

@GabrielCoderz Perspective does not support internationalization yet (`en-us` only today), and when it does it will not be a developer API nor user configurable (it will use the browser locale).

**Reply by texodus** [reply]
#2565 adds localization support and a hilariously bad Portuguese translation file
